jp/src/test/java/edu/hillel/lesson24/DatabaseStorageTest.java

```diff
@@ -0,0 +1,49 @@
+package edu.hillel.lesson24;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import org.junit.After;
+import org.junit.Before;
+import java.io.FileNotFoundException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class DatabaseStorageTest {
+    private StorageFactory storageFactory;
+    private Connection connection;
+
+    @Before
+    public void setUp() {
+        try {
+            connection = DriverManager.getConnection("jdbc:mysql://localhost:3306/testing","root","root");
```

Comment:
If somebody try to build your project this test will be failed

Reply:
Changed